Packs/CheckPhish/Integrations/CheckPhish/CheckPhish_test.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto
+import CheckPhish
+from CommonServerPython import DBotScoreReliability
+
+RESULTS = {
+    'status': 'DONE',
+    'job_id': 'jobid1234',
+    'url': 'http://test.com/',
+    'brand': 'unknown',
+    'url_sha256': '33f57dd894c986dadb1f0c9bde11bea035833560d889607bf63971093abf0695',
+    'disposition': 'clean',
+    'error': False
+}
+
+
+def test_reliability_in_get_result_checkphish(requests_mock, mocker):
+
+    requests_mock.post('https://developers.checkphish.ai/api/neo/scan/status', json=RESULTS)
+
+    mocker.patch.object(demisto, 'results')
+    CheckPhish.unite_dispositions('adult', 'cryptojacking', 'drug_spam')
+    CheckPhish.get_result_checkphish('jobid1234', 'apikey', 'https://developers.checkphish.ai/api/neo/scan', False,
+                                     DBotScoreReliability.B)
+
+    assert demisto.results.call_args_list[0][0][0]['Contents'] == RESULTS
+    assert demisto.results.call_args_list[0][0][0]['EntryContext']['DBotScore']['Reliability'] == DBotScoreReliability.B
```

Comment:
this UT is great. need to add one more to show that if `clean` disposition is added to the `bad_disp`, than the reputation is `Bad`. Let me know if i'm missing anything.

Reply:
done.